src/services/CertificateGenerationService.ts

```diff
@@ -921,9 +1007,13 @@ class CertificateGenerationService {
           defects.MinorDefects.push(this.formatDefect(defect));
           if (type === CERTIFICATE_DATA.PASS_DATA && isWelsh) {
             // TODO - remove this once tested
```

Comment:
Can the todo's be acted upon / removed

Reply:
Please leave in todo's/logs until corresponding doc-gen changes / work has been completed. They are required to assist with ongoing development.